apps/web/cypress/tests/layout/main-nav.spec.ts

```diff
@@ -85,33 +85,25 @@ describe('Main Nav (Sidebar)', () => {
       cy.getByTestId('side-nav-settings-team-link').should('exist');
       cy.getByTestId('side-nav-settings-branding-link').should('exist');
       cy.getByTestId('side-nav-settings-billing-link').should('exist');
-
-      // TODO: remove these lines when we implement toggles for different envs
-      cy.getByTestId('side-nav-settings-api-keys').should('exist');
-      cy.getByTestId('side-nav-settings-inbound-webhook').should('exist');
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please - reinstate these as they verify the buttons are there in their current form‏

Reply:
Good catch. Messed up in the rebase